services/github/github-hacktoberfest.service.js

```diff
@@ -59,66 +59,71 @@ const queryParamSchema = Joi.object({
 }).required()
 
 module.exports = class GithubHacktoberfestCombinedStatus extends GithubAuthV4Service {
-  static get category() {
-    return 'issue-tracking'
+  static category = 'issue-tracking'
+  static route = {
+    base: 'github/hacktoberfest',
+    pattern: ':year(2019|2020)/:user/:repo',
+    queryParamSchema,
   }
 
-  static get route() {
-    return {
-      base: 'github/hacktoberfest/2019',
-      pattern: ':user/:repo',
-      queryParamSchema,
-    }
-  }
-
-  static get examples() {
-    return [
-      {
-        title: 'GitHub Hacktoberfest combined status',
-        namedParams: {
-          user: 'snyk',
-          repo: 'snyk',
-        },
-        staticPreview: this.render({
-          suggestedIssueCount: 12,
-          contributionCount: 8,
-          daysLeft: 15,
-        }),
-        documentation,
+  static examples = [
+    {
+      title: 'GitHub Hacktoberfest combined status',
+      namedParams: {
+        year: '2020',
+        user: 'snyk',
+        repo: 'snyk',
       },
-      {
-        title:
-          'GitHub Hacktoberfest combined status (suggestion label override)',
-        namedParams: {
-          user: 'tmrowco',
-          repo: 'tmrowapp-contrib',
-        },
-        queryParams: {
-          suggestion_label: 'help wanted',
-        },
-        staticPreview: this.render({
-          suggestedIssueCount: 12,
-          contributionCount: 8,
-          daysLeft: 15,
-        }),
-        documentation,
+      staticPreview: this.render({
+        suggestedIssueCount: 12,
+        contributionCount: 8,
+        daysLeft: 15,
+      }),
+      documentation,
+    },
+    {
+      title: 'GitHub Hacktoberfest combined status (suggestion label override)',
+      namedParams: {
+        year: '2020',
+        user: 'tmrowco',
+        repo: 'tmrowapp-contrib',
       },
-    ]
-  }
+      queryParams: {
+        suggestion_label: 'help wanted',
+      },
+      staticPreview: this.render({
+        year: '2020',
+        suggestedIssueCount: 12,
+        contributionCount: 8,
+        daysLeft: 15,
+      }),
+      documentation,
+    },
+  ]
 
-  static get defaultBadgeData() {
-    return {
-      label: 'hacktoberfest',
-      color: 'orange',
-    }
-  }
+  static defaultBadgeData = { label: 'hacktoberfest', color: 'orange' }
 
-  static render({ suggestedIssueCount, contributionCount, daysLeft }) {
+  static render({
+    suggestedIssueCount,
+    contributionCount,
+    daysLeft,
+    daysToStart,
+    year,
+    hasStarted = true,
+  }) {
+    if (!hasStarted) {
+      return {
+        message: `${daysToStart} ${maybePluralize(
+          'day',
+          daysToStart
+        )} til kickoff!`,
```

Comment:
This spelling caught my eye. Reading on various Internet resources, it seems that `till` should be preferred.

https://www.grammarly.com/blog/until-till-til/
https://writingexplained.org/til-or-till-difference
https://www.merriam-webster.com/words-at-play/should-you-use-until-or-till-or-til

Reply:
TIL I learned that til... :laughing: